timber/src/main/java/timber/log/Timber.java

```diff
@@ -502,16 +502,16 @@ public void log(int priority, Throwable t) {
       prepareLog(priority, t, null);
     }
 
-    /** Return whether a message at {@code priority} should be logged. */
-    protected boolean isLoggable(int priority) {
+    /** Return whether a message at {@code priority} or {@code tag} should be logged. */
+    protected boolean isLoggable(int priority, String tag) {
```

Comment:
Make this two methods so it's not breaking. Have the two argument one call the single argument one.